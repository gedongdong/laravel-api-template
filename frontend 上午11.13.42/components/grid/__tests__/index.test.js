import { mount } from '@vue/test-utils';
import { Col, Row } from '..';
import mountTest from '../../../tests/shared/mountTest';

describe('Grid', () => {
  mountTest(Row);
  mountTest(Col);
  it('should render Col', () => {
    const wrapper = mount(Col, {
      props: {
        span: 2,
      },
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
  it('should render Row', () => {
    const wrapper = mount(Row);
    expect(wrapper.html()).toMatchSnapshot();
  });

  it('renders wrapped Col correctly', () => {
    const wrapper = mount({
      render() {
        return (
          <Row gutter={20}>
            <div>
              <Col span={12} />
            </div>
            <Col span={12} />
          </Row>
        );
      },
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
