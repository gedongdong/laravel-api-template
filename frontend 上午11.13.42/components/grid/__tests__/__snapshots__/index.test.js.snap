// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid renders wrapped Col correctly 1`] = `
<div class="ant-row" style="margin-left: -10px; margin-right: -10px;">
  <div>
    <div class="ant-col ant-col-12" style="padding-left: 10px; padding-right: 10px;">
      <!---->
    </div>
  </div>
  <div class="ant-col ant-col-12" style="padding-left: 10px; padding-right: 10px;">
    <!---->
  </div>
</div>
`;

exports[`Grid should render Col 1`] = `
<div class="ant-col ant-col-2">
  <!---->
</div>
`;

exports[`Grid should render Row 1`] = `
<div class="ant-row">
  <!---->
</div>
`;
